Gutenberg/Anatomy of a block.md

```diff
@@ -0,0 +1,55 @@
+New core editor is a powerful tool that gives you the ability to utilize the power of blocks. Blocks are a basic structure of your website.
+
+Because the new editor is in a phase of rapid development, at the time of writing this page (May 2020), it's known to make some breaking changes. This is why we opted out for creating our own block library and project library, to serve as an interface between the fast changing system.
+
+Since we use dynamic blocks, we wanted to find a way to reuse logic and components in blocks, to avoid having to write tons of boilerplate code, and to avoid breakage during development - which usually happens with standard blocks.
+
+## Blocks folder
+
+All Eightshift blocks are contained in the `src/blocks` folder. More in developer documentation about that folder can be found [here](https://infinum.github.io/eightshift-docs/docs/guides/blocks-structure).
+
+There are few key pieces of information to grasp.
+
+### manifest.json
+
+This is the key file in any block, as well as the overall project settings. We have one in the `blocks` folder, but also in the each block element, that are located in the `custom` folder.
+
+This file is used so that certain key information about the block is located in one place.
+
+### components
+
+Components, in our case, mimic React components - they are code pieces that can be reused in multiple blocks.
+
+### custom
+
+This is the folder where all project blocks live. One block can reuse multiple components (depending on its complexity)
+
+### assets
+
+This is a folder where JS and SCSS registration specific to blocks lives. It's where the code for registering blocks is held, but also where you can place any JS or SCSS related to blocks only.
```

Comment:
```suggestion
This is a folder where JS and SCSS registration specific to blocks lives. It's where the code for registering blocks is held, but also where you can place any JS or SCSS related to blocks only.

Note that all block / component specific JS / SCSS should be added to that specific block / component folder rather than here.
```